pkg/cmd/roachtest/github.go

```diff
@@ -98,14 +119,18 @@ func (g *githubIssues) shouldPost(t test.Test) (bool, string) {
 }
 
 func (g *githubIssues) createPostRequest(
```

Comment:
minor: why enumerate all parameters instead of just passing `(t Test, message string)`?

Reply:
ah so the `start` and `end` times belong to `*testImpl`; thus, those had to be passed in.

- the other thought was to change the `test.Test` type to `*testImpl` since `createPostRequest` only gets used with arguments of type `*testImpl` currently.
    - however, the conclusion to this (after speaking with Tobias) is that `*testImpl` is a "grab bag of stuff" that would lead to a more difficult time testing.
    
as for the `name` and `spec` fields, i did this since the function itself didn't seem to specifically need a `Test` other than to get the `name` and the `spec` (since t.Fatalf is now replaced). i am open to discuss this!

i believe this is one more parameter in the function signature than i promised Tobias...

Reply:
Make sense - I tend not to like functions with long arg lists.